kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/preprocess/SinkRecordFilter.java

```diff
@@ -0,0 +1,103 @@
+package com.wepay.kafka.connect.bigquery.preprocess;
+
+import com.jayway.jsonpath.*;
```

Comment:
usually it's cleaner to explicitly import all the classes you need to avoid unneeded collisions. It might just be an IDE setting that converted this to `*`, so check your IDE config.